### PR TITLE
Move changes into Production

### PR DESCRIPTION
**Pull request by okaycj**
### Summary

These are the changes on staging that we're going to move to production.

### CODE REVIEW

**Review by ianchandlercampbell — APPROVED** [review]
Approved pending quality checks and any additional code review (if needed) by devs! I'll write up a quick release notes later today

**Review by becky-gilbert — APPROVED** [review]
I've just done an additional code review. Looks great, thanks @okaycj! 
Let me know if there's anything that needs manual testing on staging - I'd be happy to help.

File: web/views.py [hunk 1]
```diff
@@ -841,3 +841,20 @@ def get_context_data(self, **kwargs):
...
 
         return context
+
+
+class FaqView(generic.TemplateView):
+    template_name = "web/faq.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        if get_language() == "ja":
```
Comment (becky-gilbert) [inline web/views.py:851]: @okaycj did you see the sentry error for this line? `get_language` is undefined. I think we need to edit the import line for translation to include `get_language`:

```
from django.utils.translation import get_language, gettext_lazy as _
```

Comment (okaycj) [reply web/views.py:851]: Woof!  I don't like this, so I changed our linter to flag this as an issue.  This change is in PR https://github.com/lookit/lookit-api/pull/1397.

Comment (becky-gilbert) [reply web/views.py:851]: I know!! 😬 I also don't get how the import line change wasn't included in the PR (I'm assuming it was present in the contributor's code but haven't checked). 
Anyway, thanks for setting up the linter to catch this.

**Pull request merged**